earth_enterprise/docker/config/rhel-7.sh

```diff
@@ -0,0 +1,24 @@
+# Input variable definitions for building and running Docker images for
+# testing RHEL 7.
+
+# Use this file in a command like:
+#
+# ```BASH
+# ( . config/rhel7.sh && ./bin/build-gee-image.sh )
+# ```
+
+export OS_DISTRIBUTION=rhel-7
+export STAGE_1_NAME=clean-clone
+#export REDHAT_USERNAME=put-your-username-here
+#export REDHAT_PASSWORD="your password here"
```

Comment:
Even if I set my password here, it still prints it on the screen:

```
ccamp@Mainframe:/opt/dev/tst/earthenterprise/earth_enterprise/docker$ ( . config/rhel-7.sh && ./bin/build-build-env-image.sh )
REDHAT_PASSWORD: xxxxxxxx
Creating named pipe: /opt/dev/tst/earthenterprise/earth_enterprise/docker/image-definition/build/tmp.eC91n/secrets/socket
Env server listening on: /opt/dev/tst/earthenterprise/earth_enterprise/docker/image-definition/build/tmp.eC91n/secrets/socket
===================
```

Reply:
:stuck_out_tongue_winking_eye: Good catch!  It's not good to leave debugging code behind.